helm-chart/renku-gateway/templates/deployment.yaml

```diff
@@ -125,6 +125,10 @@ spec:
               value: {{ .Values.jupyterhub.clientId | quote }}
             - name: KEYCLOAK_URL
               value: {{ .Values.keycloakUrl | default (printf "%s://%s" (include "gateway.protocol" .) .Values.global.renku.domain) | quote }}
+              {{ if .Values.global.keycloak.realm }}
```

Comment:
If the realm is set to something other than `Renku`, should we not change the name in the realm we inject into keycloak?

Reply:
Yes, I agree. For me the post-install job for keyclaok should
- create a realm with the chosen name if it doesn't exist already
- add the necessary clients to the realm of the chosen name if they do not exist already

This should be implemented when removing the realm template and using the API instead. For now this is just a small change on the gateway level which does not break need changes / break the renku chart.

Reply:
but using a different name for the realm will break the deployment.

Reply:
Sure, if that realm doesn't exist. But why would anybody do that as long as this option is not described in the main renku chart? For now and just by itself that PR doesn't change much, except allowing for some later changes to the Renku chart and enabling a use case where you have a previously existing realm and would like to use it instead of creating a new one.

Reply:
Sure -- but then please document the value clearly and with a strong warning that if using the renku-deployed keycloak it should not be changed.

Reply:
👍 done!